avm/res/automation/automation-account/credential/main.bicep

```diff
@@ -0,0 +1,42 @@
+metadata name = 'Automation Account Credentials'
+metadata description = 'This module deploys Azure Automation Account Credentials.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Automation Account credential.')
+param credentialName string
+
+@description('Conditional. The name of the parent Automation Account. Required if the template is used in a standalone deployment.')
+param automationAccountName string
+
+@description('Required. The user name associated to the credential.')
+param userName string
+
+@description('Required. Password of the credential.')
+@secure()
+param password string
+
+@description('Optional. Description of the credential.')
+param credentialDescription string = ''
```

Comment:
Same comment as for the `name`. The `credential` just seems redundant 😉

Reply:
description parameter updated

Reply:
unluckily not: introducing the user defined type

Reply:
implemented